testdata/data-transport-layer/configMap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: data-transport-layer
+data:
+  DATA_TRANSPORT_LAYER__ETH_NETWORK_NAME: hardhat
+  DATA_TRANSPORT_LAYER__NODE_PACKAGE_VERSION: "0.5.47"
+  DATA_TRANSPORT_LAYER__L1_START_HEIGHT: "1"
+  DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT: "http://l1chain-svc:8545"
+  DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT: "http://l2geth-svc:8545"
+  DATA_TRANSPORT_LAYER__L2_CHAIN_ID: "17"
```

Comment:
Okay, we have to give each chain_id to user for each network. I think we can fix it later!

Reply:
thank you!